xla/stream_executor/cuda/cuda_dnn.h

```diff
@@ -77,12 +77,21 @@ class CudnnGraph : public dnn::DnnGraph {
     current_dropout_rng_offset_[local_device_ordinal] +=
         dropout_rng_offset_increment_;
   }
+  absl::StatusOr<bool> SupportsExplicitCommandBufferConstruction()
+      const override;
+  absl::Status PopulateOrUpdateRawCommandBuffer(
+      Stream&, absl::Span<DeviceMemoryBase> operands, RawCommandBufferHandle,
+      bool do_update) override;
 
  private:
   cudnn_frontend::graph::Graph graph_;
   int64_t dropout_rng_seed_;
   mutable std::vector<int64_t> current_dropout_rng_offset_;
   int64_t dropout_rng_offset_increment_ = 0;
+  using VariantPack = std::unordered_map<int64_t, void*>;
```

Comment:
ClangTidy:

```
The types in this container are small enough such that references to the underlying objects are unlikely to exist. Thus pointer stability is extraordinarily unlikely to be required. Consider using absl::flat_hash_set or absl::flat_hash_map!
```

Reply:
unordered_map<int64_t, void*> is the data type taken by the methods of the cuDNN frontend: populate_cuda_graph(), update_cuda_graph(), execute().